package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "@whereby.com/browser-sdk",
-    "version": "2.0.0-alpha13",
+    "version": "2.0.0-alpha14",
```

```suggestion
    "version": "2.0.0-alpha15",
```
14 is already released now